src/main/java/com/smartbear/mqttsupport/connection/dialog/EditConnectionDialog.java

```diff
@@ -684,13 +687,19 @@ private void activateTab(String tabName) {
         }
     }
 
-    private boolean checkCertificateField(JTextField edit) {
-        if (StringUtils.isNullOrEmpty(edit.getText())) {
+    private String expandProperty(JTextField edit) {
```

Comment:
I think it will be more sufficient to pass the text itself as an argument instead of the JTextField as the only thing we do with that field is `edit.getText()`. It's a good practice to pass the most specific type that your method needs to do its job. If your method only needs the text from the JTextField, then it's better to just pass the text as a String